geoprocessing/src/utils/file.utils.ts

```diff
@@ -25,3 +29,36 @@ export const uploadOptions: Readonly<MulterOptions> = {
     },
   }),
 };
+
+export class FileService {
```

Comment:
Looks like it should be `Injectable()` and wrapped in some module instead

Reply:
@kgajowy 1d15b56c1ef8c605a970984b80f2616a35a65f64